ui/src/Service.js

```diff
@@ -21,19 +24,32 @@ export class Service {
     getPlayerId = () => {
         const request = new GeneratePlayerIdRequest();
 
-        const metadata = this.getMetadata();
-
-        return this.playerIdServicePromiseClient.generatePlayerId(request, metadata);
+        return this.playerIdServicePromiseClient.generatePlayerId(request, this.getMetadata());
     }
 
     getTopPlayerScore = () => {
         const request = new TopScoresRequest();
 
         request.setSize();
 
-        const metadata = this.getMetadata();
+        return this.leaderboardServicePromiseClient.getTopScores(request, this.getMetadata());
+    }
+
+    subscribeOnTopScoreStream = (handler) => {
```

Comment:
Does it mean you created this branch off the leadearboard streaming branch? Streaming leaderboard changes haven't been merged yet - this piece have to be taken out and rebased to the corresponding branch.

Reply:
My changes branch was branched off of streaming-and-unary-leaderboard